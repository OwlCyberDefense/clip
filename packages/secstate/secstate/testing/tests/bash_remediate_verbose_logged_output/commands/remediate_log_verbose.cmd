/usr/sbin/secstate remediate -y -v -l /root/remediation.logv
