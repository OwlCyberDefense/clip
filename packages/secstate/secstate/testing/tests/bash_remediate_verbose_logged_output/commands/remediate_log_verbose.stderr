-- remediating password-req --
running bash remediation script: /root/passreqs.sh
running bash remediation script: /root/passreqs.sh
running bash remediation script: /root/passreqs.sh
running bash remediation script: /root/passreqs.sh
running bash remediation script: /root/passreqs.sh
running bash remediation script: /root/passreqs.sh
running bash remediation script: /root/passreqs.sh
running bash remediation script: /root/passreqs.sh
running bash remediation script: /root/passreqs.sh
